### PR TITLE
added list component

### PR DESCRIPTION
**Pull request by DivyanshSareen**
Added List component

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.list-item{
+    padding: var(--padding-s);
+}
+
+/*-- Stacked List --*/
+/* simply added some border and shadow to spaced list */
+.stacked-list>.list-item{
+    box-shadow: var(--list-shadow);
```
Comment (iRohitGaur) [inline components/list.css:23]: This will not work. Please check once

Comment (DivyanshSareen) [reply components/list.css:23]: sorry ig the preview link broke, can you check it on this one https://deploy-preview-8--peaceui.netlify.app/list.html

Comment (iRohitGaur) [reply components/list.css:23]: I am talking about your box-shadow. Please check: https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

File: components/style-guide.css [hunk 2]
```diff
@@ -78,6 +78,8 @@
   /* shadows used */
   --button-shadow: 2px 2px 5px rgb(0 0 0 / 0.5);
   --card-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+  /* yup card and list use the same shadow */
+  --list-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
```
Comment (srishtigautam21) [inline components/style-guide.css]: Use var for colors

**Review by srishtigautam21 — APPROVED** [review]
Do the changes suggested by rohit otherwise code looks fine

File: components/list.css (hunk 1 above)
Comment (DivyanshSareen) [reply components/list.css:23]: I have defined the variable like this `--list-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24); ` , do let me know if the variable name is misleading, will replace it

**Review by itsmebipul00 — APPROVED** [review]

File: components/list.css (hunk 1 above)
Comment (iRohitGaur) [reply components/list.css:23]: Okay. Makes sense now 👍🏻

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Pull request merged**